app/graphql/types/customers/usage/charge.rb

```diff
@@ -8,13 +8,18 @@ class Charge < Types::BaseObject
 
         field :amount_cents, GraphQL::Types::BigInt, null: false
         field :events_count, Integer, null: false
+        field :id, ID, null: false
         field :units, GraphQL::Types::Float, null: false
 
         field :billable_metric, Types::BillableMetrics::Object, null: false
         field :charge, Types::Charges::Object, null: false
         field :filters, [Types::Customers::Usage::ChargeFilter], null: true
         field :grouped_usage, [Types::Customers::Usage::GroupedUsage], null: false
 
+        def id
+          SecureRandom.uuid
```

Comment:
I think just `Random.uuid` would be good enough here? SecureRandom has some overhead to make sure it's secure, is that needed for this use-case?

Reply:
Well, I just need a unique ID for each items, it does not matter if the ID changes every time I fetch the data.

Sounds like `Random.uuid` can potentially create twice or more the same ID?
This object is received in an array with potentially many others.

Reply:
it's not possible for Random.uuid to generate the same ID. However, my concerns were based on outdated information and it shouldn't matter 👍